### ISSUE TITLE
Sourcing R code

### ISSUE THREAD

**Issue by yteo** [Azure/AzureStor#135]
Switching from AWS to Azure and trying to understand if there is an equivalence of [s3source](https://www.rdocumentation.org/packages/aws.s3/versions/0.3.21/topics/s3source) in the AzureStor package? I am trying to source an R code directly from a blob. Thanks!

**Comment by hongooi73** [comment]
No, but you can download to a temp file and source that. Here's a simple function to do this and clean up afterward:

```
azure_source <- function(cont, file)
{
  dest <- tempfile(fileext=".R")
  on.exit(unlink(dest))
  storage_download(cont, file, dest)
  source(dest)
}
```
